utils/sccli.py

```diff
@@ -0,0 +1,224 @@
+                service_stop(service_name)
+                break
+        return status
+    else:
+        return system("systemctl is-active %s" % service_name)[2]
+
+def service_restart(service_name):
+    if service_status(service_name):
+        return ('Start %s first' % service_name, 110)
+    if service_name == "kubernetes":
+        return system("systemctl restart etcd kube-apiserver",
+                      " kube-controller-manager kube-scheduler",
+                      " kube-proxy kubelet docker")[1:]
+    else:
+        return system("systemctl restart %s" % service_name)[1:]
+
+
+def service_stop(service_name):
+    if service_name == "kubernetes":
+        system("systemctl disable etcd kube-apiserver "
+               "kube-controller-manager kube-scheduler kube-proxy kubelet")
+        if not system("systemctl is-active kube-apiserver")[2]:
+            system("systemctl stop kube-apiserver")
+        if not system("systemctl is-active kube-controller-manager")[2]:
+            system("systemctl stop kube-controller-manager")
+        if not system("systemctl is-active kube-scheduler")[2]:
+            system("systemctl stop kube-scheduler")
+        if not system("systemctl is-active kube-proxy")[2]:
+            system("systemctl stop kube-proxy")
+        if not system("systemctl is-active kubelet")[2]:
+            system("systemctl stop kubelet")
+        system("systemctl reset-failed kube-apiserver"
+               " kube-controller-manager kube-scheduler kube-proxy kubelet")
+        if not system("systemctl is-active etcd")[2]:
+            system("systemctl stop etcd")
+    elif service_name == "openshift":
+        system("systemctl disable openshift")
+        if not system("systemctl is-active openshift")[2]:
+            system("systemctl stop openshift")
+        system("systemctl reset-failed openshift")
+    elif service_name == "docker":
+        if not system("systemctl is-active docker")[2]:
+            system("systemctl stop docker")
+        system("systemctl reset-failed docker")
+
+def setup_kube_service_account_key():
+    system("mkdir -p /etc/pki/kube-apiserver/")
+    system("openssl genrsa -out /etc/pki/kube-apiserver/serviceaccount.key 2048")
+    system("sed -i.back '/KUBE_API_ARGS=*/c\KUBE_API_ARGS="
+           "\"--service_account_key_file=/etc/pki/kube-apiserver/serviceaccount.key\"'"
+           " /etc/kubernetes/apiserver")
+    system("sed -i.back '/KUBE_CONTROLLER_MANAGER_ARGS=*/c\KUBE_CONTROLLER_MANAGER_ARGS="
+           "\"--service_account_private_key_file=/etc/pki/kube-apiserver/serviceaccount.key\"'"
+           "/etc/kubernetes/controller-manager")
+
+# This is not used "need discussion"
+def vagrant_box_variant():
+    try:
+        with open("/etc/os-release") as fh:
+            for line in fh.readlines():
+                if "Container Development Kit" in line:
+                    return False
+        return True
+    except IOError as err:
+        return (err, 37)
```

Comment:
@LalatenduMohanty it's not used yet but in case we don't want to pull images again for openshift  in case of CDK then we can use it. I already commented on that function.